### PR TITLE
Craftify

### PR DESCRIPTION
**Pull request by KaranBengani**
# Application for Registering Team to CODE CAMP 1.0

This pull request template helps you complete an application to the **CODE CAMP 1.0 HACKATHON**. Use the checklist below to verify that you have followed the instructions correctly. 

**(Put x inside [] like [x])**

## Select the type of Pull Request

- [x] Registration Pull request.

[] Submission Pull request.

## Checklist ✅  

- [x] I have read the instructions on the README file before submitting my application and gone through [CODE_OF_CONDUCT](). 
- [x] I have Submitted my team's details by making a folder of my team as instructed in [How to send Pull Request page](https://codecamphackathon.netlify.app/makingRegistration/).
- [x] I have used the Markdown file template to add my information for the Hackathon.
- [x] I have given all the necessary details as mentioned in template index.md file and the details are correct and best to my knowledge.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the Details added to the template will be used as a means of communication at the time of result declaration.
- [x] I have added the [event](https://calendar.google.com/calendar/r/eventedit/copy/MHUzM2pvaDQwM3R1cDdxMmdjMmQyYjZpaHAgN3NhMzI0N2FpMzkzczAzbWo0YWMxNWVzdG9AZw) to my calendar.

### CODE REVIEW

**Comment by Uyadav207** [comment]
@KaranBengani , Have a look at your **Description** : DO not include any sort of special symbols write in words..

The way you should go with: 

Description: "Here Goes your Teams description"

File: content/teams/Code Blooded/Index.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+---
+type: "team"                                                        
+title: Code Blooded
+name: "Code Blooded"
+github-link: "https://github.com/KaranBengani "
+description: "While("CODE BLOODED"=="BEST TEAM")
+               { printf("we will win");}"
```
Comment (Uyadav207) [inline content/teams/Code Blooded/Index.md]: @KaranBengani , Have a look at your Description : DO not include any sort of special symbols write in words..

The way you should go with:

Description: "Here Goes your Teams description"

Comment (Uyadav207) [reply content/teams/Code Blooded/Index.md]: One line of description is sufficient.

Comment (KaranBengani) [reply content/teams/Code Blooded/Index.md]: i have updated it, i hope its fine now

**Comment by Uyadav207** [comment]
@KaranBengani , Hola  👋 ! Team  Code Blooded have successfully completed the `First phase of the Hackathon`. ( We have successfully merged your PR for Registration ). 
Now we Request you to join your Team to our [**CODE CAMP GROUP 1**](https://chat.whatsapp.com/EowU6RVas3QCmtEs7q7L0H).
And you can start making your project from `21th July,2020`.
We have attached important Links. 

[**TimeLine**](https://codecamphackathon.netlify.app/TimeLine/)
[**Rule Book**](https://codecamphackathon.netlify.app/RuleBook/)
[**Prizes**](https://codecamphackathon.netlify.app/Prizes/)

See you there!
@ISTESRMNCR

**Comment by KaranBengani** [comment]
all members need to join the whatsapp group?

**Pull request merged**